topics/webapp/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: web-application
+display_name: Webapp
+short_description: A web application is any computer program that performs a specific function by using a web browser as its client.
+topic: webapp
+related: angular, perl, xml, html
```

Comment:
These seem like pretty general topics, not specifically related to web apps. What about [app](https://github.com/topics/app), [mobile-app](https://github.com/topics/mobile-app), [webdev](https://github.com/topics/webdev)?